### DISCUSSION TITLE
Does this work on Steam Deck?

### DISCUSSION THREAD

**Discussion by RaymondYouTube** [coqui-ai/TTS#4059] [General]
If you don't know what it is, the Steam Deck is a handheld Linux gaming computer made by Valve. The Steam Deck uses an AMD Custom APU, it also uses SteamOS which is based on Arch Linux. Anyway does Coqui-TTS work on Steam Deck? I just wanna use it to make videos just for craps and giggles.

**Comment by eginhard** [comment]
You can try installing [our fork](https://github.com/idiap/coqui-ai-TTS) (`pip install coqui-tts`) which has better cross-platform support. Let me know how it goes. The GPU won't be supported though for sure.
